CorsixTH/Lua/entities/humanoids/staff.lua

```diff
@@ -626,7 +627,7 @@ function Staff:updateDynamicInfo()
     dynamic_text,
     fatigue_text,
   })
-  self:setDynamicInfo('progress', self.attributes["fatigue"])
+  self:setDynamicInfo('progress', self.attributes.fatigue or 0)
```

Comment:
This is not suitable as it creates a tiredness bar on a receptionist
![image](https://github.com/CorsixTH/CorsixTH/assets/20030128/e419ce1c-9775-4bb1-8e47-3395abf76289)

Reply:
(You can likely either add a case to ignore receptionists, or the old check may be fine as is. See `Entity:setDynamicInfo`)

Reply:
Reverted this too.